templates/default/delos.css

```diff
@@ -1,4 +1,7 @@
 @charset "UTF-8";
+/*
```

Comment:
There are a lot of changes in this file, and I can't figure out why :-). The changed lines should only contain chatroom-based changes.